src/components/PostComment.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import { postComment } from "../utils/api";
+import moment from "moment";
+
+export const PostComment = ({ review_id }) => {
+  const [body, setBody] = useState("");
+  let username = "jessjelly";
+
+  const handleSubmit = (e) => {
+    if (body.length < 10 || username === "") {
+      e.preventDefault();
+      alert("Comments must be 10 characters or more");
+    } else {
+        document.getElementById("comment-body").disabled = true;
+      postComment(review_id, username, body).then(
+        ( comment ) => {
+            alert("Comment posted")
+          return (
+            <>
+          <div className="comment-card">
+            <p>Username: {comment.author}</p>
+            <p>{comment.body}</p>
+            <p>Votes: {comment.votes}</p>
+            <p>{moment(comment.created_at).format("LLL")}</p>
+          </div>
```

Comment:
returning this JSX from the promise won't do anything!

only JSX returned from the React component function will be rendered on the page